### ISSUE TITLE
Restructure Settings Pages

### ISSUE THREAD

**Issue by PriPatel** [lightdash/lightdash#5420]
**Blocked by** https://github.com/lightdash/lightdash/issues/5457

https://www.figma.com/proto/gtLzWNBwSX1ejmwz7MEm4W/Untitled?type=design&node-id=211-12869&scaling=min-zoom&page-id=106%3A13594&starting-point-node-id=211%3A12869

- [ ] Add 'Current Project' section in the sidebar and move out separate Project pages under there

**Comment by IrakliJani** [comment]
@PriPatel extracted [Change the components on the 'All Projects' page to a table format #5681](https://github.com/lightdash/lightdash/issues/5681) as a separate ticket
